### ISSUE TITLE
add audio files to apkg file

### ISSUE THREAD

**Issue by yakhinvadim** [yakhinvadim/longman-to-anki#27]

**Comment by yakhinvadim** [comment]
I've tried using google text-to-speech while learning cards and found it difficult to focus on text, while someone is speaking. So, I guess text-only version will be ok.

**Comment by yakhinvadim** [comment]
I think we can at least try it. I guess if it irritates someone, they could turn it off in the Anki app.

**Comment by omega3** [comment]
Exactly. Anki has in deck preferences an option "Automatically play audio" and this can be unchecked. 
A lot of users want to have audio. This is why text to speech add-on is for, but original Longman audio is a lot better than google.

I am in favor of adding audio files for sentences.

**Comment by yakhinvadim** [comment]
Thank you for your thoughts, I think I'll do it. I don't want to promise any exact dates, but this will be the first thing in my list when I have some free time.
